recipes/mbdata/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mbdata" %}
+{% set version = "26.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mbdata-{{ version }}.tar.gz
+  sha256: a0ada9081b29f72502aebb859589cfed39aac91eed7e6f944da9e90a3e5c547a
+
+build:
+  entry_points:
+    - mbslave=mbdata.replication:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - contextlib2  # [py<32]
+    - python
+    - psycopg2
+    - six
+    - typing  # [py<3]

```

Comment:
We only build >=3.6 (3.7 soon) so I wonder if it is worth dropping `contextlib2` and `typing` and make this noarch?

Reply:
You were right, even on pypi this is a tarball instead of specific wheels for each versions and arch.
https://pypi.org/project/mbdata/26.0.1/#files